src/python/zquantum/core/interfaces/backend.py

```diff
@@ -4,9 +4,10 @@
     create_bitstring_distribution_from_probability_distribution,
 )
 from ..circuit import Circuit, CircuitConnectivity
-from ..measurement import ExpectationValues, Measurements
+from ..measurement import ExpectationValues, Measurements, expectation_values_to_real
+from ..openfermion import expectation
```

Comment:
I think name of this function is a little bit too vague and not a verb.
`get_expectation_from_wavefunction` or sth like this?

Reply:
I agree, but I would suggest making a chore for this and addressing in a future PR since it will require unrelated  changes that need to be propogated